README.md

```diff
@@ -11,7 +11,7 @@
 Clone the GitHub repository
 ```bash
 git clone https://github.com/astrolabsoftware/fink-client.git
-cd fink-client
+cd fink-client && export FINK_CLIENT_HOME=$(PWD)
```

Comment:
you could even suggest to permanently store this in e.g. the `.bash_profile` of the user to avoid typing this again and again.

Reply:
Resolved in [`e85b4a0`](https://github.com/astrolabsoftware/fink-client/pull/22/commits/e85b4a0150c7881dd656c9e436ac6efd2c046925)

I can re-word this further if you like, I'm not sure if I explained things properly